tests/layer_tests/pytorch_tests/test_all.py

```diff
@@ -64,24 +93,26 @@ def test_all_noparams(self, input_shape, d_type, ie_device, precision, ir_versio
         ([10, 13, 11], np.int64),
         ([8, 7, 6, 5, 4], np.int64),
         ([11, 11], np.uint8),
-        ([7, 7], np.uint8)
+        ([7, 7], np.uint8),
+        ([4, 4], bool)
     ])
     @pytest.mark.parametrize("keepdim", [
         True,
         False,
         None
     ])
+    @pytest.mark.parametrize("out", [True, False])
     @pytest.mark.nightly
     @pytest.mark.precommit
     @pytest.mark.xfail(condition=platform.system() in ('Darwin', 'Linux') and platform.machine() in ('arm', 'armv7l',
                                                                                                      'aarch64',
                                                                                                      'arm64', 'ARM64'),
                        reason='Ticket - 122715')
-    def test_all(self, input_shape, d_type, keepdim, ie_device, precision, ir_version):
+    def test_all(self, input_shape, d_type, keepdim, out, ie_device, precision, ir_version):
```

Comment:
Everything is perfect, only suggestion is to change `d_type` to just `dtype`, I have never seen this varable name split like this